util/Pos/Util/CompileInfo.hs

```diff
@@ -52,7 +53,7 @@ withCompileInfo = give
 retrieveCompileTimeInfo :: TH.Q TH.Exp
 retrieveCompileTimeInfo = do
     cti <- TH.runIO $ do
-      ctiGitRevision <- fromString <$> retrieveGit
+      ctiGitRevision <- T.strip . fromString <$> retrieveGit
```

Comment:
Strip removes space characters, but not newline character (which is problem here)

Reply:
@akegalj My experience is different with `strip` (I have used it no problem in other projects):

![screen shot 2017-12-18 at 09 48 56](https://user-images.githubusercontent.com/442035/34097246-baf56cc4-e3d8-11e7-8ab9-d82cb4b146f8.png)

Am I missing something here?

Note that `Data.Text.strip` is implemented in terms of `isSpace` from `Data.Char`, which documentation states:

```
Returns True for any Unicode space character, and the control characters \t, \n, \r, \f, \v.
```

Reply:
@adinapoli ah, you are right. I didn't read documentation of `isSpace` and that newline is considered a space:
```haskell
Prelude> import Data.Char
Prelude Data.Char> isSpace '\n'
True
```

sorry for the confusion.